cmd/k8sm-executor/main.go

```diff
@@ -154,18 +153,15 @@ func main() {
 	standalone.RunKubelet(&kcfg, builder)
 
 	log.Infoln("Waiting for driver initialization to complete")
-	<-initialized
-	defer driver.Destroy()
 
 	// block until driver is shut down
-	driver.Join()
+	select {}
```

Comment:
Interesting trick.  Will that cause the driver to block instead of driver.Join() ?